SPECS/openldap/openldap.spec

```diff
@@ -15,9 +14,12 @@ Patch1:         openldap-2.4.44-consolidated-2.patch
 Patch2:         CVE-2015-3276.patch
 Patch3:         CVE-2021-27212.patch
 Patch4:         CVE-2022-29155.patch
+%global _default_patch_fuzz 2
```

Comment:
Please move this back to the top of the spec file - the linter is not doing a great job with the macros, unfortunately.